src/components/App/App.jsx

```diff
@@ -31,6 +31,69 @@ export default function App() {
     quantity: ""
   }
 
+  var sampleRecipes = [
```

Comment:
nit: Same comment as on the other PR, you could add a TODO comment here. Since this data came from a specific API, it might also be worth adding a comment with the source of this data as well both to remind yourself (and in theory other people working on this code)

Reply:
Got it! I added the TODO comment